### PR TITLE
Feature/share access

### PR DESCRIPTION
**Pull request by andrewhick**
Expanded the register feature to include granting access to agents to view licences, as well as Browserstack support and changes to match latest UI.

There is one more puts statement which I believe to be helpful to anyone executing or maintaining the tests - this helps a user see which licence holder and agent was used as part of each test.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Into that file you'll need to add as a minimum this
 # immediately failing because the element cannot be found. This defaults to 2
 # seconds but with the need to confirm emails via mailinator, we have found we
 # need to increase this time to at least 5 seconds
-max_wait_time: 5
+max_wait_time: 10
```
Comment (timstone) [inline README.md]: Did you mean to do this in the readme?

File: Rakefile [hunk 2]
```diff
@@ -16,11 +16,97 @@ end
 
 desc "Run all scenarios (eq to bundle exec quke)"
 task :run do
-  sh %( bundle exec quke )
+  sh %( QUKE_CONFIG=.config.yml bundle exec quke )
```
Comment (timstone) [inline Rakefile]: Default is that config file anyway isn't it?

File: Rakefile [hunk 3]
```diff
@@ -16,11 +16,97 @@ end
...
+#
+# To run Browserstack tests, use bundle exec rake bs
+
+desc "Run all browser tests"
+task bs: %i[ios_safari]
+# separate browsers with spaces
+
+# rubocop:disable Metrics/LineLength
+# Text for task bs: win7_ie9 win7_ie11 win7_chrome win7_ff win10_edge macos_safari macos_chrome macos_ff ios_safari android_chrome samsung
+# rubocop:enable Metrics/LineLength
```
Comment (timstone) [inline Rakefile]: Do you need these commented out bits?

File: Rakefile [hunk 4]
```diff
@@ -16,11 +16,97 @@ end
...
+# win7_ff (v58) pass
+# win10_edge pass
+# macos_safari pass
+# macos_chrome (v64) pass
+# macos_ff (v58) pass
+# ios_safari FAIL
+# ios_chrome PASS MANUAL
+# android_chrome pass
+# samsung (Galaxy S8) pass
+# winphone_ie pass MANUAL
```
Comment (timstone) [inline Rakefile]: This is a task file not a test management tool! ;)

File: README.md (hunk 1 above)
Comment (andrewhick) [reply README.md]: No - well spotted :)

File: Rakefile (hunk 4 above)
Comment (andrewhick) [reply Rakefile]: Fair point :)

**Pull request merged**